content/sponsors/geocat.md

```diff
@@ -42,4 +42,4 @@ publishing data from your desktop GIS with a single click). For an even more str
 
 We also offer expert consultancy and training to help you get the most out of our products.
 
-Want to learn more? Contact us at [info@geocat.net](mailto:info@geocat.net) or visit [www.geocat.net](https://www.geocat.net).
+Want to learn more? Contact us at [info@geocat.net](mailto:info@geocat.net) or visit <a href="https://www.geocat.net" target="_blank" rel="noopener noreferrer">www.geocat.net</a>.

```

Comment:
please use hugo's `extlink` shortcode for any links pointing to external sites:

`{{ extlink href="https://www.geocat.net" title="www.geocat.net" }}`


(see comment at [PR#18](https://github.com/foss4g-europe/2025-foss4g-europe.github.io/pull/18#discussion_r2005209518))